src/github.ts

```diff
@@ -1,24 +1,70 @@
 import * as github from "@actions/github";
+import * as core from '@actions/core'
+import { IParameter } from './issue'
 
-export const getPrNumber = (): number | undefined => {
-  const pullRequest = github.context.payload.pull_request;
-  if (!pullRequest) {
-    return undefined;
+export interface IRepo {
+  owner: string;
+  repo: string;
+}
+
+export class GithubApi {
+  private octokit;
+  private repo: IRepo
+  private issueNumber: number | undefined
+
+  constructor(token: string) {
+    this.octokit = new github.GitHub(token)
+    this.repo = github.context.repo
+    if(github.context.payload.issue) this.issueNumber = github.context.payload.issue.number
```

Comment:
Should be separated lines with `{}` usage or same alignment.

```suggestion
    if(github.context.payload.issue) this.issueNumber = github.context.payload.issue.number
    else if (github.context.payload.pull_request) this.issueNumber = github.context.payload.pull_request.number
    else core.setFailed(`Error retrieving issue number`)
```

or

```suggestion
    if(github.context.payload.issue) {
    	this.issueNumber = github.context.payload.issue.number
    } else if (github.context.payload.pull_request) {
    	this.issueNumber = github.context.payload.pull_request.number
    } else {
    	core.setFailed(`Error retrieving issue number`)
  	}
```